index.js

```diff
@@ -14,8 +14,20 @@ function Geoservices () {}
 function pullDataAndRoute (model, req, res) {
   model.pull(req, function (error, data) {
     if (error) {
-      const err = normalizeError(error)
-      res.status(err.code || 500).json({ error: err.message })
+	    if (!error.error) {
```

Comment:
Can we move all of this to helper function (same file is fine), e.g, `sendError(req, res, error)`?  It would be nice to abstract away all the details.